plover/assets/main.json

```diff
@@ -721,7 +721,7 @@
 "-R": "are",
 "-RB": "{,}",
 "-RB/-RB/TPHR-RB": "{,}",
-"-RB/TPHR-RB/TK-RB": " -- ",
+"-RB/TPHR-RB/TK-RB": "--",
```

Comment:
I think we should just remove this one instead of fixing it, since according to @stenoknight:
> And you can ignore all strokes that contain TPHR-RB. Those are legacy strokes from Eclipse's terrible time-based buffer system, but fortunately Plover doesn't have to deal with that.